plugins/spi/SPIOutput.h

```diff
@@ -38,6 +38,22 @@ namespace spi {
 
 class SPIOutput: public ola::rdm::DiscoverableRDMControllerInterface {
  public:
+  // definitions for all SPI Personalities
+  // this hardcoded mapping is needed for backwards compatibility
+  // enum class SPI_PERSONALITY : unsigned int {
+  enum SPI_PERSONALITY {
+    PERS_WS2801_INDIVIDUAL = 1,
+    PERS_WS2801_COMBINED = 2,
```

Comment:
I believe if you specify the value for the first item, the rest will increment afterwards, so not hardcoding the other values and just adding a note to keep this order may be clearer?